pom.xml

```diff
@@ -56,6 +56,24 @@
       <artifactId>google-cloud-vision</artifactId>
       <version>1.70.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
+      <artifactId>powermock-module-junit4</artifactId>
+      <version>1.6.4</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
+      <artifactId>powermock-api-mockito</artifactId>
+      <version>1.6.4</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Is there a reason you're adding powermock twice in dependencies?

Reply:
That's what the documentation says.